rfcs/react-components/convergence/semantic-shared-colors.md

```diff
@@ -0,0 +1,195 @@
+
+| Current name | Current meaning       | Proposed change                                                                                                                     |
+| ------------ | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `red`        | danger, busy, persona | Introduce semantic `danger`, mapped to `cranberry` instead of `red`.<br/>Keep `red`, it is still used for **busy** and **persona**. |
+| `green`      | success               | Replace by semantic `success`.                                                                                                      |
+| `darkOrange` | severe                | **severe** should not be used anywhere. Keep `darkOrange` (temporarily) for Teams compatibility.                                    |
+| `yellow`     | warning               | Replace by semantic `warning`, mapping changed from `yellow` to `orange`                                                            |
+| `berry`      | OOF                   | No change                                                                                                                           |
+| `lightGreen` | available             | No change                                                                                                                           |
+| `marigold`   | away, persona         | No change                                                                                                                           |
+
+Note: even though the task now is to introduce the semantic names only for the **status** tokens, in the future we might be asked to do the same change for **presence** tokens.
+
+### Other platforms
+
+I briefly checked react native packages for iOS and Android produced by token pipeline, they use tokens like `warningBackground1`.
+
+## Detailed Design or Proposal
+
+There are multiple options how to address the problem. This chapter describes the options and their pros and cons.
+
+### 1. Introduce new semantic status tokens
+
+Add 3 new semantic colors:
+
+1. danger
+2. success
+3. warning
+
+For each of the 3 new colors, add 9 (or 10) color slots (see the table above).
+
+Update the sources in the library to use the semantic color names instead of the current shared color names.
+
+Find a way how to get rid of the legacy tokens which are no longer needed:
+
+- `green` (10 tokens)
+- `yellow` (10 tokens)
+
+Once the legacy tokens are not used by the library code, there are three possible ways how to get rid of the tokens:
+
+#### 1A: "deprecate" the tokens and remove them in the next major release
+
+#### 1B: "deprecate" the tokens and create a V2 theme which does not contain them - let applications switch when they are ready
```

Comment:
If I correctly understand it:

````suggestion
#### 1B: "deprecate" the tokens and create a V2 theme which does not contain them - let applications switch when they are ready

```ts
// @deprecated Use tokensV2
const tokens = {
  // old tokens to be deprecated
  // @deprecated Use tokensV2.colorStatusSuccessBackground1
  colorPaletteGreenBackground1: 'var(--colorPaletteGreenBackground1'),
  // @deprecated Use tokensV2.colorStatusWarningBackground1
  colorPaletteYellowBackground1: 'var(--colorPaletteYellowBackground1'),
}
const tokensV2 = {
  // new tokens (note: naming is not final)
  colorStatusDangerBackground1: 'var(--colorStatusDangerBackground1)',
  colorStatusWarningBackground1: 'var(--colorStatusWarningBackground1)', 
}

const webLightTheme = {
  colorPaletteGreenBackground1: 'HEX_VALUE',
  colorPaletteYellowBackground1: 'HEX_VALUE',
}
const webLightThemeV2 = {
  colorStatusDangerBackground1: 'HEX_VALUE',
  colorStatusWarningBackground1: 'HEX_VALUE', 
}
```
````

IMO, it might be easier to have "-v2" package at this point.